### PR TITLE
Hello, I modified the the readme

### PR DESCRIPTION
**Pull request by C15klodins**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ The Python Home Budget App is a beginner-friendly project aimed at those who are
...
 
 Team member
-1. CN
-2. KM
-3. SA
-4. BB
+1. CN, a teacher
+2. KM, a teacher
+3. SA, a student
+4. BB, a student
```
Comment (NyergesC) [inline README.md:21]: i am also a teacher

Comment (C15klodins) [reply README.md:21]: Awesome, great to meet you

**Review by 1Belda — CHANGES_REQUESTED** [review]
Dear team, 
I do not approve on this change I need to review and read Solomon work as he is the only student in this class 
 Best regards,

**Review by NyergesC — APPROVED** [review]
great job! 💯

**Comment by KindsonTheGenius** [comment]
> great job! 💯

LGTM

**Pull request merged**